Code/src/main/java/nl/utwente/group10/ui/handlers/ConnectionCreationManager.java

```diff
@@ -0,0 +1,91 @@
+package nl.utwente.group10.ui.handlers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+import java.util.Optional;
+
+import javafx.geometry.Point2D;
+import javafx.scene.Parent;
+import nl.utwente.group10.ui.CustomUIPane;
+import nl.utwente.group10.ui.components.Connection;
+import nl.utwente.group10.ui.components.ConnectionAnchor;
+import nl.utwente.group10.ui.components.ConnectionLine;
+import nl.utwente.group10.ui.components.InputAnchor;
+import nl.utwente.group10.ui.components.OutputAnchor;
+
+public class ConnectionCreationManager {
```

Comment:
Ik weet het niet zo goed.
Hij houd bij welke input bij welke lijn hoort, en hoe die lijnen zich dus moeten gedragen.
Ik vind het dan toch wel meer een Manager dan een Factory of Builder